modules/aws_eks/aws-eks.yaml

```diff
@@ -57,7 +57,7 @@ inputs:
     default: "t3.medium"
   - name: k8s_version
     user_facing: true
-    validator: enum("1.19", "1.20", "1.21", "1.22", required=False)
+    validator: enum("1.19", "1.20", "1.21", "1.22", "1.23", "1.24", "1.25", required=False)
     description: The Kubernetes version for the cluster. Must be [supported by EKS](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
     default: "1.21"
```

Comment:
While ppdating this default to EKS' default of `1.25` sounds nice, it would cause diffs in existing deployments, if a value was not explicitly specified.

Reply:
It's explicitly set in all of our deployments to 1.22...